src/Modal.svelte

```diff
@@ -470,6 +477,7 @@
       style={cssWindowWrap}
     >
       <div
+        id={state.id}
```

Comment:
To be more generic, the ID should be assigned to the root DOM element. I.e., the background `<div />`. Alternatively, we could also add ID props for each element like we have for the class properties. However, this seems unnecessary as one can easily query for a specific modal element if one has the ID of the root element and the class names.

E.g.,

```svelte
<script>
	import Content from './Content.svelte';
  import Modal from './Modal.svelte';
	import Popup from './Popup.svelte';
	import { modal } from './stores.js';

  const id = 'my-modal';
  const classWindow = 'my-window';

	modal.subscribe((value) => {
		if (!value) return;
		requestAnimationFrame(() => {
			console.log(document.querySelector(`#${id} .${classWindow}`));
		})
	});
</script>

<Modal show={$modal} id={id} classWindow={classWindow}>
	<Content />
</Modal>
```